modules/web/src/assets/config/changelog.json

```diff
@@ -1,149 +1,21 @@
     },
     {
       "category": "fixed",
-      "description": "Fix a bug where dedicated credentials were incorrectly being required as mandatory input when editing provider settings for a cluster."
+      "description": "Cluster-autoscaler addon now works based on the namespace instead of cluster names; all MachineDeployments in the `kube-system` namespace are scaled."
     },
     {
       "category": "fixed",
-      "description": "Fix invalid project ID in API requests for Nutanix provider."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix a bug where OSPs were not being listed for VMware Cloud Director."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix issue in editing and updating applications of cluster template."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix issue with cursor position inside YAML editor."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix a bug where dedicated credentials were incorrectly being required as mandatory input when editing provider settings for a cluster."
-    },
-    {
-      "category": "fixed",
-      "description": "No longer fail constructing vSphere endpoint when a `/` suffix is present in the datacenter configuration."
-    },
-    {
-      "category": "fixed",
-      "description": "Resolved an issue where logs were duplicated when multiple pods from the same service were deployed on the same Kubernetes node."
-    },
-    {
-      "category": "fixed",
-      "description": "Stop constantly re-deploying operating-system-manager when registry mirrors are configured."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix OIDC network policy, by allowing to set NamespaceOverride to specify where the ingress controller is deployed."
-    },
-    {
-      "category": "fixed",
-      "description": "Fix missing image registry override for hubble-ui components if Cilium is deployed as System Application."
-    },
-    {
-      "category": "fixed",
-      "description": "Enable web terminal button when at least one MD replica is ready."
-    },
-    {
-      "category": "deprecated",
-      "description": "Deprecate v1.11 and v1.12 Cilium and Hubble KKP Addons, as Cilium CNI is managed by Applications from version 1.13."
+      "description": "Add Canal CNI version v3.27.3, having a fix to the ipset incompatibility bug."
     },
     {
-      "category": "removed",
-      "description": "Some of high cardinality metrics were dropped from the User Cluster MLA Prometheus. If your KKP installation was using some of those metrics for the custom Grafana dashboards for the user clusters, your dashboards might stop showing some of the charts."
+      "category": "security",
+      "description": "Update Cilium to 1.14.9 and 1.13.14, mitigating [CVE-2024-28860](https://github.com/advisories/GHSA-pwqm-x5x6-5586) and [CVE-2024-28248](https://github.com/advisories/GHSA-68mj-9pjq-mc85)."
```

Comment:
I don't think this supports markdown (note to myself: remove the links).

Reply:
Fixed!